cmake/modules/FindSQLite3.cmake

```diff
@@ -0,0 +1,16 @@
+message("Looking for SQLite3...")
+find_path(SQLITE3_INCLUDE NAMES sqlite3.h)
+mark_as_advanced(SQLITE3_INCLUDE)
+if(SQLITE3_INCLUDE)
+	include_directories(${SQLITE3_INCLUDE})
+endif()
+
+find_library(SQLITE3_LIBRARY NAMES sqlite3)
+mark_as_advanced(SQLITE3_LIBRARY)
+if(SQLITE3_LIBRARY)
+	add_library(sqlite3 SHARED IMPORTED)
+	set_property(TARGET sqlite3 PROPERTY IMPORTED_LOCATION "${SQLITE3_LIBRARY}")
+endif()
+
+include(FindPackageHandleStandardArgs)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(SQLite3 DEFAULT_MSG SQLITE3_LIBRARY SQLITE3_INCLUDE)
```

Comment:
Could you set your editor to include newlines at the end of files, so Git doesn't get annoyed?

Reply:
Yeah, sorry about that. I'm going through now to fix missing newlines